lib/alchemy/page_layout.rb

```diff
@@ -83,9 +83,14 @@ def layouts_with_own_for_select(page_layout_name, language_id, only_layoutpages
       #   language_id of current used Language.
       # @param [Boolean] (false)
       #   Pass true to only select layouts for global/layout pages.
+      # @param [String]
+      #   name of current/parent page layout
       #
-      def selectable_layouts(language_id, only_layoutpages = false)
+      def selectable_layouts(language_id, only_layoutpages = false, page_layout_name = nil)
         @language_id = language_id
+        @page_layout = page_layout_name.present? ? all.detect{ |layout| layout['name'] == page_layout_name } : nil
+        return [] if @page_layout.present? && @page_layout['sub_pages'] == false
```

Comment:
Here as well, use `present?` to account for `nil` values

Reply:
Again, this would mean that all page layouts are required to specify their sub_pages, while i only intended this as an optional restriction.

Reply:
Oh yeah, we wouldn't want that. This needs to be absolutely optional.